ttnctl/cmd/devices_set.go

```diff
@@ -204,5 +213,6 @@ func init() {
 	devicesSetCmd.Flags().Int32("altitude", 0, "Set altitude")
 
 	devicesSetCmd.Flags().String("description", "", "Set Description")
-	devicesSetCmd.Flags().StringArray("custom-attr", []string{}, "Set a custom attribute")
+	devicesSetCmd.Flags().StringArray("attr-add", []string{}, "Add an additional attribute (ex:--attr-add <name>:<value>)")
```

Comment:
I think attribute is the right naming. Indeed, people can put random stuff, but that's up to the people.

We may want to restrict the key to match our generic regex for ids? That way, we can safely use it in the future